pyproject.toml

```diff
@@ -22,7 +22,7 @@ install_requires = [
 artifact_id = "navx-frc-cpp"
```

Comment:
should this have bumped the wpilib lower bound too?

Reply:
Yes.